state.go

```diff
@@ -0,0 +1,70 @@
+// +build linux
+
+package main
+
+import (
+	"os"
+	"time"
+
+	"encoding/json"
+
+	"github.com/codegangsta/cli"
+)
+
+// cState represents the platform agnostic pieces relating to a running
+// container's status and state.  Note: The fields in this structure adhere to
+// the opencontainers/specs requirement for json fields that must be returned
+// in a state command.
+type cState struct {
+	// Version is the OCI version for the container
+	Version string `json:"ociVersion"`
+	// ID is the container ID
+	ID string `json:"id"`
+	// InitProcessPid is the init process id in the parent namespace
+	InitProcessPid int `json:"pid"`
+	// Bundle is the path on the filesystem to the bundle
+	Bundle string `json:"bundlePath"`
+	// Path to a directory containing the container's root filesystem.
```

Comment:
I initially miss-read your request as to the output text not the // comment.  I've updated both now in a second refresh..